cli/js/web/streams/internals.ts

```diff
@@ -2371,8 +2344,7 @@ function writableStreamMarkFirstWriteRequestInFlight<W>(
 ): void {
   assert(stream[sym.inFlightWriteRequest] === undefined);
   assert(stream[sym.writeRequests].length);
-  const writeRequest = stream[sym.writeRequests].shift();
-  stream[sym.inFlightWriteRequest] = writeRequest;
+  stream[sym.inFlightWriteRequest] = stream[sym.writeRequests].shift();
```

Comment:
There are some strange code in here, because it follows the spec statement by statement.  A lot of the logic shortcuts I don't have a problem with, but this one specifically deviates from the spec as far as how things are assigned.  I would prefer we don't adjust this.